packages/ui/src/i18n/dictionaries/authenticator/en.ts

```diff
@@ -14,6 +14,7 @@ export const enDict = {
   Confirming: 'Confirming',
   'Create Account': 'Create Account',
   'Creating Account': 'Creating Account',
+  'Dismiss alert': 'Dismiss alert',
```

Comment:
Do we need to add key/values for our other dictionaries (example: _i18n/dictionaries/authenticator/id.ts_) to make sure we are covering international customers?

Reply:
It would be nice, but we don't have the translations for other languages yet. We could add them in a future PR. If you look at the translations now, there is no parity between the translations we offer from one language to another.

Reply:
Perhaps I can look into translations of those other texts, in this PR.

Reply:
I'll request that Totoro translate this text